src/SonarQube.Client/Api/V6_30/GetPluginsRequest.cs

```diff
@@ -29,6 +30,9 @@ namespace SonarQube.Client.Api.V6_30
 {
     public class GetPluginsRequest : RequestBase<SonarQubePlugin[]>, IGetPluginsRequest
     {
+        private const string sonarQubeVersionPattern = @"(?<major>\d+)[\.]?(?<minor>\d+)?[\.]?(?<patch>\d+)?.*";
+        private static readonly Regex SonarVersionRegEx = new Regex(sonarQubeVersionPattern, RegexOptions.None, TimeSpan.FromMilliseconds(300));
```

Comment:
The `SonarQube.Client` class doesn't reference `Core`, and it can't at the moment because it is reference by `Core` -> I couldn't use the new timeout constant here.